ApiBundle/Controller/ContentController.php

```diff
@@ -113,6 +113,12 @@ public function listAction(Request $request)
         );
 
         $repository =  $this->get('open_orchestra_model.repository.content');
+
+        if ($request->get('entityId') && $request->get('language') && $request->get('version')) {
```

Comment:
By doing this, the method return is not consistent, I would rather have an collection with only one element inside.

What do you think?